src/utilities/theme/tokens.ts

```diff
@@ -13,6 +13,10 @@ const Shadow = {
   topBarShadow: '0 2px 2px -1px var(--p-shadow-from-direct-light)',
 };
 
+const RGB = {
+  rgbText: '33, 43, 54',
+};
+
```

Comment:
Just thinking out-loud in this PR but maybe we should think about exposing RGB color values from our factory

Reply:
I'm going to remove this from the PR and look into adding into `polaris-tokens` unless someone thinks that might be unnecessary.

Reply:
Removed for now. I don't think it makes sense to include these in this PR. I'll explore adding these to tokens at some point